tests/openadapt/test_recording.py

```
@@ -3,15 +3,133 @@
 import multiprocessing
 import time
 import os
+import signal
 import pytest
+import psutil
```

Ruff

`psutil` imported but unused

Remove unused import: `psutil`

(F401)